ShareX.UploadersLib/ShareX.UploadersLib.csproj

```diff
@@ -795,11 +794,9 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   <PropertyGroup>
-    <PreBuildEvent>cd $(ProjectDir)APIKeys\
-
-if not exist APIKeysLocal.cs (
-    type nul &gt; APIKeysLocal.cs
-)</PreBuildEvent>
+    <PreBuildEvent>cd $(ProjectDir)APIKeys
+echo=&gt;&gt; APIKeysLocal.cs
```

Comment:
This is just trying to append (add to existing or create new if none exists) a new line to the APIKeysLocal.cs file.  I can add a file check if wanted.